setup.py

```diff
@@ -100,12 +99,16 @@ def run(self):
 hdfs = ["pyarrow>=2.0.0;  python_version < '3.9'"]
 webhdfs = ["hdfs==2.5.8"]
 webdav = ["webdavclient3>=3.14.5"]
+osf = ["osfclient==0.0.4"]
 # gssapi should not be included in all_remotes, because it doesn't have wheels
 # for linux and mac, so it will fail to compile if user doesn't have all the
 # requirements, including kerberos itself. Once all the wheels are available,
 # we can start shipping it by default.
 ssh_gssapi = ["paramiko[invoke,gssapi]>=2.7.0"]
-all_remotes = gs + s3 + azure + ssh + oss + gdrive + hdfs + webhdfs + webdav
+all_remotes = (
+    gs + s3 + azure + ssh + oss + gdrive + hdfs + webhdfs + webdav + osf
```

Comment:
I wonder if at some point we'll have to do multiple bundles? DVC can become too heavy.

Reply:
But you can use ``pip install dvc[osf]`` to use dvc with the selected remote storage. I suppose that ```all_remotes``` is mainly needed to install the development and testing environment.

Reply:
you can. The problem that we have brew, deb, win, dmg, and a few other ways we deliver this. And those include all the dependencies.